tests/test_job.py

```diff
@@ -55,7 +55,7 @@ def builtins_dict():
 
 def config_from_cfg(cfg):
     cfile = io.StringIO("\n".join(cfg))
-    return signac.common.config.get_config(cfile)
+    return signac.common.config._get_config(cfile)
```

Comment:
Rewrite this helper to use functions from the public API for 2.0 if possible.

Reply:
The reason I didn't do this is listed in the description: those changes are already made in next so this will likely just lead to conflicts that I don't want to address again :)

Reply:
Sorry! I misunderstood which conflicts you were describing. This is fine as-is, then.